data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -73,7 +93,18 @@
       ]
     }
   ],
-  "contributor": [
-    "array of strings (CV)"
-  ]
+  "meta": {
```

Comment:
This is the draft of a 'meta' field also added to person and work records most recently.